docs/ocis/adr/0001-introduce-accounts-service.md

```diff
@@ -1,4 +1,10 @@
-# 1. Introduce an accounts service
+---
+title: "1. Introduce an accounts service"
+date: 2020-06-15T20:21:00+01:00
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/adr
```

Comment:
must be `geekdocEditPath: edit/master/docs/adr`

Reply:
but the file lives in `docs/ocis/adr`?

Reply:
sorry did not see that they are moved in this PR

Reply:
sorry did not see that they are moved in this PR